_layouts/integration.html

```diff
@@ -73,6 +86,36 @@ <h2>API support</h2>
   </details>
 </section>
 
+{% if page.quality_estimation_integrations.size > 0 %}
+<section id="qe-integrations">
+  <h2>Quality estimation</h2>
+  <details>
+    <summary>
+      <strong>{{ page.quality_estimation_integrations| size }}</strong> Quality estimation API integrations are available for {{ page.title }}.
+      <p class="preview hint">
+        {{page.quality_estimation_integrations | slice: 0, 5 | map: 'name' | join: ', ' }}
+        {% if page.quality_estimation_integrations > 5 %}, …{% endif %}
+      </p>
+    </summary>
+    <ul>
+    {% for qe in page.quality_estimation_integrations %}
+      <li>
+        {% if qe.plugin %}
+          <a href="/{{ qe.slug }}">{{ qe.name }}</a> (plugin)
+        {% else %}
+          <a href="/{{ qe.slug }}">{{ qe.name }}</a>
+        {% endif %}
+          {% if qe.custom or qe.glossary or qe.formality %}
```

Comment:
Glossary and formality are not properties of formality.

Really we need an official schema for our JSON in _data.

You could create an issue for that.

@cefoo